docs/reference/github-intro.md

```diff
@@ -0,0 +1,8 @@
+# Using Git, GitHub and GitHub Desktop (version control)
+
+A repository can consist of many files with several users simultaneously editing those files at any moment in time. In order to ensure conflicting edits between the users are not made and a history of the edits are tracked, a distributed version control system is used. 
```

Comment:
I think "distributed version control system" is a bit much for someone needing this document. (totally understand that it will take a bit to explain and doesn't belong in reference hence my above comment

Reply:
It's not showing here, but if you go to the "Files changed" tab, it shows the sentence updated to:
In order to ensure conflicting edits between the users are not made and a history of the edits are tracked, software classified as a "distributed version control system" is used.

I thought it would be useful to include the term 'distributed version control system' since the user may see it mentioned elsewhere whilst learning Git and at least would know it's a description of the software, even if not fully grasping what it means.

Reply:
yeah this works :) thanks